tools/ci_build/github/linux/docker/cuda_centos6.repo

```diff
@@ -0,0 +1,6 @@
+[cuda]
+name=cuda
+baseurl=http://developer.download.nvidia.com/compute/cuda/repos/rhel6/x86_64
```

Comment:
>baseurl [](start = 0, length = 7)

@snnn Is this the right base?

Reply:
Can you reuse this one: https://github.com/microsoft/onnxruntime/blob/master/tools/ci_build/github/linux/docker/cuda_manylinux2010.repo ?

Reply:
Yes, I copied from there.

---
In reply to: [336222556](https://github.com/microsoft/onnxruntime/pull/2159#discussion_r336222556) [](ancestors = 336222556)

Reply:
But do we need two?